notebooks/Emukit-tutorial-custom-model.ipynb

```diff
@@ -11,6 +11,9 @@
    "cell_type": "markdown",
```

Comment:
<p>That's a rubbish change</p>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>